Assets/Prefabs/Object/Asteroid.prefab

```diff
@@ -94,10 +94,14 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 9be033a3997445ce822eb199ad904517, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  minMoveSpeed: 0.1
-  maxMoveSpeed: 0.12
```

Comment:
![ask](https://img.shields.io/badge/review-ask-9b71ae.svg) 
Is this change intended? I think we decided to go with this number 0.1 - 0.12.
(But of course we can make changes after this PR merged)

Reply:
same as above